bin/MadGraph5_aMCatNLO/cards/WZ_leptonic/WLLjj_dim8_ewk_madspin_card.dat

```diff
@@ -0,0 +1,26 @@
+#************************************************************
+#* MadSpin *
+#* *
+#* P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer *
+#* *
+#* Part of the MadGraph5_aMC@NLO Framework: *
+#* The MadGraph5_aMC@NLO Development Team - Find us at *
+#* https://server06.fynu.ucl.ac.be/projects/madgraph *
+#* *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+#initialization parameters
+#set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+set BW_cut 15 # cut on how far the particle can be off-shell
+#set max_weight_ps_point 400 # number of PS to estimate the maximum for each event
+#
+set max_running_process 1
+
+decay w+ > l+ vl
+decay w- > l- vl~
+decay z > l+ l-
+# running the actual code
+launch
```

Comment:
hello, i sent an email, please remove this file

Reply:
Hi, okay I have removed this file.